src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -48,22 +48,22 @@ export const hasRequiredConsents = (): Promise<boolean> =>
 type PreCheckArgs = {
     brazeSwitch: boolean;
     apiKey?: string;
-    isDigitalSubscriber?: boolean;
+    shouldHideSupportMessaging?: boolean;
     pageConfig: { [key: string]: any };
 };
 
 export const canShowPreChecks = ({
     brazeSwitch,
     apiKey,
-    isDigitalSubscriber,
+    shouldHideSupportMessaging,
```

Comment:
As with the Frontend version, it might be nice to document our intent here, so that the reader can see whether `shouldHideSupportMessaging` meets that requirement now and in the future?